### ISSUE TITLE
Need some UI indicator when transition between py2 and py3 in notebook.

### ISSUE THREAD

**Issue by qimingj** [googledatalab/datalab#1606]
If you switch kernels between py2 and py3 from UI, it takes time for the transition but there is no indication what is going on in UI. Usually it is about 5 sec on my desktop but it could be even longer in n1-standard-1.

**Comment by nikhilk** [comment]
We should also try to determine how important it is to switch within a notebook, vs. 

- picking it at time of creation
- a more global setting to pick the default (as I think many customers will try to standardize on one)

This has bearing on the ux.

**Comment by qimingj** [comment]
+1 on picking it at creation time so we don't need to support switching.

**Comment by chmeyers** [comment]
jimmc made some changes around this in https://github.com/googledatalab/datalab/pull/1575

Are those changes sufficient?

**Comment by qimingj** [comment]
Oh. I did not see that. I think it is sufficient.

But we should revisit Nikhil's point on whether we need the switch?

**Comment by yebrahim** [comment]
Yes. Let's rename the issue and keep it or create a new one. Picking a default or creating a notebook with a certain kernel both sound like very desirable features.
